README.md

```diff
@@ -10,10 +10,11 @@ Analogous to the Core Team for Swift, the work group is responsible for providin
 
 The current Swift Server work group consists of the following people:
 
-* Gwynne Raskind ([@gwynne](https://github.com/gwynne), Vapor)
```

Comment:
I don't think we want to remove Gwynne right?

Reply:
Updated